stack/dashboard/deb/build_package.sh

```diff
@@ -35,51 +35,51 @@ ctrl_c() {
 }
 
 build_deb() {
-    CONTAINER_NAME="$1"
-    DOCKERFILE_PATH="$2"
+    container_name="$1"
+    dockerfile_path="$2"
 
     # Copy the necessary files
-    cp ${CURRENT_PATH}/builder.sh ${DOCKERFILE_PATH}
+    cp ${current_path}/builder.sh ${dockerfile_path}
 
     # Build the Docker image
-    if [[ ${BUILD_DOCKER} == "yes" ]]; then
-        docker build -t ${CONTAINER_NAME} ${DOCKERFILE_PATH} || return 1
+    if [[ ${build_docker} == "yes" ]]; then
+        docker build -t ${container_name} ${dockerfile_path} || return 1
     fi
 
 
     # Build the Debian package with a Docker container
-    VOLUMES="-v ${OUTDIR}/:/tmp:Z"
-    if [ "${REFERENCE}" ];then
-        docker run -t --rm ${VOLUMES} \
-            ${CONTAINER_NAME} ${ARCHITECTURE} ${REVISION} \
-            ${FUTURE} ${BASE} ${REFERENCE} || return 1
+    volumes="-v ${outdir}/:/tmp:Z"
+    if [ "${reference}" ];then
+        docker run -t --rm ${volumes} \
+            ${container_name} ${architecture} ${revision} \
+            ${future} ${base} ${reference} || return 1
     else
-        if [ "${BASE}" = "local" ];then
-            VOLUMES="${VOLUMES} -v ${BASE_PATH}:/root/output:Z"
+        if [ "${base}" == "local" ];then
```

Comment:
Same

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1648/commits/88b143bb8737d8b65fb27be9fe101805a0fe3d7a